### DISCUSSION TITLE
Welcome to ZettelGeist Discussions!

### DISCUSSION THREAD

**Discussion by gkthiruvathukal** [ZettelGeist/zettelgeist#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dwbcampbell** [comment]
I'm Douglas Campbell, I'm a software developer in Waterloo, Ontario. I have a B.Math from Waterloo from long ago, and five years ago I  completed a B.A. in Philosophy through the University of London distance program. 

I heard about the Zettelkasten method last year and have been using it for my notes on my reading in philosophy. I was using [Zettlr](https://www.zettlr.com) for a while but I don't like being tied to a special purpose editor, since I would normally be using emacs. (Another project I looked at was [Neuron] (https://neuron.zettel.page), which is used from the command line, but the installation process is a little heavyweight.) I went googling around and discovered zettelgeist. I've installed it and used it but I haven't tried the search database yet. 

I see mention of iOS on the Zettelgeist page. I've done a lot of development on iOS and would be glad to help out.  I'm currently doing C++/QT for a cross-platform app but it's not a lot of fun. :-(
